release.nix

```diff
@@ -0,0 +1,25 @@
+let
+
+  config = {
+    packageOverrides = pkgs: rec {
+      haskellPackages = pkgs.haskell.packages.ghc822.override {
```

Comment:
Wrong GHC version.

Reply:
Is there a source of truth for current versioning information that I can refer to in the future? Perhaps something in the cardano-sl repo?